src/OrbitGl/TimeGraphLayout.cpp

```diff
@@ -101,8 +101,8 @@ float TimeGraphLayout::GetCollapseButtonSize(int indentation_level) const {
 
 float TimeGraphLayout::GetBottomMargin() const {
```

Comment:
We should probably get rid of the layout elements that are no longer meaningful - just a comment, not needed in this PR.